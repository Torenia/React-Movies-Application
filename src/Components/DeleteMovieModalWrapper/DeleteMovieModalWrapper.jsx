import styled from 'styled-components';

export const DeleteMovieModalWrapper = styled.div`
  width: 600px;
  height: 300px;
  background-color: #232323;
  
  p {
   color: #fff;
   padding: 20px 80px;
  }
  
  button {
   position: relative;
   left: 300px;
   top: 40px;
  }
`;
