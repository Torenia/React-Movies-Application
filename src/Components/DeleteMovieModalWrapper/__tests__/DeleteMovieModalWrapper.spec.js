import React from 'react';

import { DeleteMovieModalWrapper } from '../DeleteMovieModalWrapper';

describe('<DeleteMovieModalWrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<DeleteMovieModalWrapper />);
  });

  it('should render a movie modal wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
