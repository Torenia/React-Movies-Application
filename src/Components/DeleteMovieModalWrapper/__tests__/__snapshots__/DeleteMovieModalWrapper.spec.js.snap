// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeleteMovieModalWrapper /> should render a movie modal wrapper 1`] = `
.c0 {
  width: 600px;
  height: 300px;
  background-color: #232323;
}

.c0 p {
  color: #fff;
  padding: 20px 80px;
}

.c0 button {
  position: relative;
  left: 300px;
  top: 40px;
  background: #f65261;
  width: 160px;
  border-radius: 5%;
}

<div
  className="c0"
/>
`;
